### PR TITLE
[internal-only] OSDOCS-10831: Documented the 4.16 Welcome Page

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16

Issue:
[OSDOCS-10831](https://issues.redhat.com/browse/OSDOCS-10831)

Link to docs preview:
[Welcome page](https://77306--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/index.html)

### CODE REVIEW

File: welcome/index.adoc [hunk 1]
```diff
@@ -151,6 +146,8 @@ xref:../installing/installing_azure/installing-azure-private.adoc#installing-aws
 xref:../installing/installing_gcp/installing-gcp-private.adoc#installing-gcp-private[{gcp-short}], or
 xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[{ibm-cloud-name}]. Internet access is still required to access the cloud APIs and installation media.
 
+- **xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-install-iscsi-manual_installing-bare-metal[Installing RHCOS manually on an iSCSI boot device] and xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-install-iscsi-ibft_installing-bare-metal[Installing RHCOS on an iSCSI boot device using iBFT]**: You can manually install {op-system} on an iSCSI boot device or on completely diskless machine where the iSCSI target and initiator values can be passed through a Boot Firmware Table (iBFT). The iBFT method supports iSCSI multipathing.
```
Comment (jldohmann) [inline welcome/index.adoc:149]:
```suggestion
- **xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-install-iscsi-manual_installing-bare-metal[Installing RHCOS manually on an iSCSI boot device] and xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-install-iscsi-ibft_installing-bare-metal[Installing RHCOS on an iSCSI boot device using iBFT]**: You can target iSCSI devices as the root disk for installation of {op-system}. Multipathing is also supported.
```

**Review by jldohmann — COMMENTED** [review]
Some minor comments 🙂

File: welcome/index.adoc (hunk 1 above)
Comment (dfitzmau) [reply welcome/index.adoc]: Thanks. Updated.

File: welcome/index.adoc [hunk 2]
```diff
@@ -72,15 +72,18 @@ Explore the following {product-title} installation tasks:
 
 - **xref:../installing/index.adoc#ocp-installation-overview[{product-title} installation overview]**: Depending on the platform, you can install {product-title} on installer-provisioned or user-provisioned infrastructure. The {product-title} installation program provides the flexibility to deploy {product-title} on a range of different platforms.
 
+// PR open https://github.com/openshift/openshift-docs/pull/77474
```
Comment (dfitzmau) [inline welcome/index.adoc:75]: Awaiting for the feature docs to be live. See https://issues.redhat.com/browse/OSDOCS-10687

File: welcome/index.adoc [hunk 3]
```diff
@@ -300,6 +287,25 @@ After configuring monitoring, use the web console to access xref:../observabilit
...
+and more.
+You can xref:../storage/expanding-persistent-volumes.adoc#expanding-persistent-volumes[expand persistent volumes], configure xref:../storage/dynamic-provisioning.adoc#dynamic-provisioning[dynamic provisioning], and use CSI to xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-using-csi[configure], xref:../storage/container_storage_interface/persistent-storage-csi-cloning.adoc#persistent-storage-csi-cloning[clone], and use xref:../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots[snapshots] of persistent storage.
+
+- **xref:../storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc[Persistent storage using CIFS/SMB CSI Driver Operator (Technology Preview)]**: {product-title} is capable of provisioning persistent volumes (PVs) with a Container Storage Interface (CSI) driver for the Common Internet File System (CIFS) dialect/Server Message Block (SMB) protocol. The CIFS/SMB CSI Driver Operator that manages this driver is in Technology Preview status.
```
Comment (tmalove) [inline welcome/index.adoc:302]:
```suggestion
- **xref:../storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc[Persistent storage using CIFS/SMB CSI Driver Operator (Technology Preview)]**: {product-title} is capable of provisioning persistent volumes (PVs) with a Container Storage Interface (CSI) driver for the Common Internet File System (CIFS) dialect/Server Message Block (SMB) protocol. The CIFS/SMB CSI Driver Operator that manages this driver is in Technology Preview status.
```
The anchor ID is missing from the xref:  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#internal-cross-references

File: welcome/index.adoc [hunk 4]
```diff
@@ -300,6 +287,25 @@ After configuring monitoring, use the web console to access xref:../observabilit
...
+
+- **xref:../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots-overview_persistent-storage-csi-snapshots[Changing vSphere CSI maximum number of snapshots]**:  The default maximum number of snapshots in VMWare vSphere Container Storage Interface (CSI) is 3 per volume. In {product-title} {product-version}, you can now change this maximum number of snapshots to a maximum of 32 per volume. You also have granular control of the maximum number of snapshots for vSAN and Virtual Volume datastores.
```
Comment (tmalove) [inline welcome/index.adoc:304]:
```suggestion
- **xref:../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots-overview_persistent-storage-csi-snapshots[Changing vSphere CSI maximum number of snapshots]**:  The default maximum number of snapshots in VMware vSphere Container Storage Interface (CSI) is 3 per volume. In {product-title} {product-version}, you can now change this maximum number of snapshots to a maximum of 32 per volume. You also have granular control of the maximum number of snapshots for vSAN and Virtual Volume datastores.
```

**Review by tmalove — APPROVED** [review]

File: welcome/index.adoc (hunk 2 above)
Comment (apinnick) [reply welcome/index.adoc:75]: This issue is at risk and will probably not be ready in time. I recommended that the PR to remove agent-based installation of Alibaba Cloud be reverted if  possible. I guess we will hear the decision tomorrow.

**Pull request merged**